keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClassicLoadBalancerHandler.kt

```diff
@@ -201,6 +276,35 @@ class ClassicLoadBalancerHandler(
         ResourceDiff(desired, current?.get(region))
       }
 
+  private fun ClassicLoadBalancerSpec.generateOverrides(
+    regionalClbs: Map<String, ClassicLoadBalancer>
+  ) =
+    regionalClbs.forEach { (region, clb) ->
+      val dependenciesDiff = ResourceDiff(clb.dependencies, dependencies).hasChanges()
+      val listenersDiff = ResourceDiff(clb.listeners, listeners).hasChanges()
+      val healthCheckDiff = ResourceDiff(clb.healthCheck, healthCheck).hasChanges()
+
+      if (dependenciesDiff || listenersDiff || healthCheckDiff) {
+        (overrides as MutableMap)[region] = ClassicLoadBalancerOverride(
+          dependencies = if (dependenciesDiff) {
```

Comment:
can you use `dependenciesDiff ?: null` here to shorten?

Reply:
`dependenciesDiff` != `clb.dependencies`